app/views/split_checkout/_voucher_section.html.haml

```diff
@@ -17,4 +17,4 @@
                 = t("split_checkout.step2.voucher.warning_forfeit_remaining_amount")
         - else
           = text_field_tag "[order][voucher_code]", params.dig(:order, :voucher_code), data: { action: "input->toggle-button-disabled#inputIsChanged", }, placeholder: t("split_checkout.step2.voucher.placeholder") , class: "voucher"
-          = submit_tag t("split_checkout.step2.voucher.apply"), name: "apply_voucher", disabled: true, class: "button cancel voucher",  data: { "toggle-button-disabled-target": "button" }, id: "testdisable"
+          = submit_tag t("split_checkout.step2.voucher.apply"), name: "apply_voucher", disabled: true, "data-disable-with": false, class: "button cancel voucher",  data: { "toggle-button-disabled-target": "button" }, id: "testdisable"
```

Comment:
I had tried that but with putting it in the `data` hash like : `data: { "toggle-button-disabled-target": "button", "disable-with": false }` but for some reason it doesn't work ? am I missing something ? 
Your suggesting does work though!

Reply:
Yeah I had to rummage around in the source code and play with it a bit, but it looks like in this case the `submit_tag` method was not nicely parsing options given in the `{ data: {"disable-with": false }` format before applying the logic for that flag, and was only disabling it when it was in the straight `"data-disable-with": false` format. I guess it's treating the former as a Hash :man_shrugging: 

Anyway, it works :tada: